operations/System_Recovery.md

```diff
@@ -0,0 +1,528 @@
+# System Recovery
+
+The following covers redeploying critical services and restoring the management service data - both from a per service perspective as well as from a fresh install perspective.
+
+The following workflows are available:
+
+* [Prerequisites](#prerequisites)
+* [System recovery of critical service](#system-recovery-of-critical-service)
+  * [Vault](#vault)
+  * [Keycloak](#keycloak)
+  * [Spire](#spire)
+  * [Nexus](#nexus)
+* [System recovery after fresh install](#system-recovery-after-fresh-install)
+
+<a name="prerequisites"></a>
```

Comment:
Please remove all of the `<a>` anchors.

The headers themselves should be cross-referenced instead of anchor refs.

Example:

```markdown

* [Prerequisites](#Prerequisites)

## Prerequisites
```

Reply:
The `<a>` anchors were used for Stash support, and were all removed in release/1.3. They only exist in release/1.2, but are still unnecessary.

Using them breaks several text editors, since they aren't native markdown.